lib/Sapi.php

```diff
@@ -67,7 +67,20 @@ static function sendResponse(ResponseInterface $response) {
             }
 
         }
-        file_put_contents('php://output', $response->getBody());
+
+        if(is_null($response->getBody())) return;
+
+        if($response->hasHeader('Content-Length')) {
+            $clen = $response->getHeader('Content-Length');
+            $output = fopen('php://output', 'wb');
+            if(is_string($response->getBody())) {
```

Comment:
I reckon it is better to test if `$response->getBody()` is a stream, then we `stream_copy_to_stream`, else we `fwrite`, because if the body is an object, we will have a strange behavior (and the `__toString` method not callable).